go.mod

```diff
@@ -27,10 +27,11 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/gruntwork-io/terratest v0.41.15
 	github.com/hoisie/mustache v0.0.0-20160804235033-6375acf62c69
+	github.com/jackc/pgx/v5 v5.3.1
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/kennygrant/sanitize v1.2.4
 	github.com/kumahq/protoc-gen-kumadoc v0.3.1
-	github.com/lib/pq v1.10.7
+	github.com/lib/pq v1.10.7 // indirect
```

Comment:
this is only here because of `cirello.io/pglock`

Reply:
Would: https://github.com/cirello-io/pglock/issues/21 solve this partly?

Reply:
done